src/Base/RestAPI/ItemFields/SeoPress.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Adds portal url field to the output.
+ */
+
+namespace OWC\OpenPub\Base\RestAPI\ItemFields;
+
+use OWC\OpenPub\Base\Models\Item;
+use OWC\OpenPub\Base\Support\CreatesFields;
+use WP_Post;
+
+class SeoPress extends CreatesFields
+{
+    /**
+     * The condition for the creator.
+     *
+     * @return callable
+     */
+    protected function condition(): callable
+    {
+        return function () {
+            return true;
+        };
+    }
+
+    /**
+     * Create the portal seopress fields for a given post.
+     *
+     * @param WP_Post $post
+     *
+     * @return string
+     */
+    public function create(WP_Post $post): array
+    {
+        $seoMeta = $this->plugin->config->get('seopress.meta');
```

Comment:
Zou hier nog even checken of dit wel een array is, de filteredMetadata is getypehint op een array vandaar.